store/ssmstore.go

```diff
@@ -559,3 +574,19 @@ func getChangeType(version int) ChangeEventType {
 	}
 	return Updated
 }
+
+func parseServiceLabel(serviceAndLabel string) (string, string) {
+	if labelMatchRegex.MatchString(serviceAndLabel) {
+		i := strings.Index(serviceAndLabel, ":")
+
+		if i > -1 {
+			service := serviceAndLabel[:i]
+			label := serviceAndLabel[i+1:]
+			return service, label
+		} else {
```

Comment:
nit: you don't need these `else` blocks since the `if` will always return

Reply:
i removed them and updated the PR